schemas/mozza/event/event.1.parquet.txt

```diff
@@ -0,0 +1,29 @@
+message event {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64  Timestamp;
+    required binary Date (UTF8);
```

Comment:
This is a header specified by the client so will only be available if the client sends it.